frontend/test/example_test.dart

```diff
@@ -12,11 +12,24 @@ import 'package:angular2/src/modules/testing/lib/testing.dart';
 import 'package:angular2/angular2.dart';
 import 'package:test/test.dart';
 
-//import 'package:frontend/event_list_component.dart';
+import 'packages/mockito/mockito.dart';
+
+import 'package:frontend/event_list_component.dart';
+import 'package:frontend/services/event_service.dart';
+import 'package:frontend/models/event.dart';
+
+import 'package:angular2/router.dart';
+import 'package:http/http.dart';
 
 @Component(selector: 'test-cmp', template: '<textarea>Hello World!\n</textarea>')
 class DummyComponent {}
 
+class EventServiceMock extends Mock implements EventService {}
+
+class RouterMock extends Mock implements Router {}
+
+class ClientMock extends Mock implements Client {}
```

Comment:
You should not need to mock the Client, that is injected through the pubspec, if I understand it correctly.
But it would only be the service which would uses it which we mock anyways.